test/torch_test_meta.py

```diff
@@ -159,60 +159,57 @@
+    'test_empty_slice', # stride
+    'test_byte_tensor_assignment', # expecting a different runtime error
+    'test_byte_mask', # expecting a different runtime error
+    'test_byte_mask_accumulate', # expecting a different runtime error
+    'test_bool_indices', # expecting a different runtime error
     'test_index_getitem_copy_bools_slices',  # storage
     'test_getitem_scalars',  # storage
-    'test_empty_ndim_index',  # runtime error
+    'test_empty_ndim_index',  # expecting a different runtime error
+
+    # NumpyTests
+    'test_trivial_fancy_out_of_bounds',  # expecting a different runtime error
+    'test_boolean_assignment_value_mismatch',  # expecting a different runtime error
     'test_empty_tuple_index',  # storage
-    'test_empty_fancy_index',  # runtime error
+    'test_empty_fancy_index',  # expecting a different runtime error
     'test_ellipsis_index',  # storage
-    'test_broaderrors_indexing',  # runtime error
-    'test_broadcast_subspace',  # runtime error
-    'test_boolean_shape_mismatch',  # runtime error
-    'test_boolean_indexing_weirdness',  # runtime error
-    'test_boolean_indexing_weirdness_tensors',  # runtime error
+    'test_broaderrors_indexing',  # expecting a different runtime error
+    'test_broadcast_subspace',  # XLA doesn't support arange with negative step
+    'test_boolean_shape_mismatch',  # expecting a different runtime error
+    'test_boolean_indexing_weirdness',  # expecting a different runtime error
+    'test_boolean_indexing_weirdness_tensors',  # expecting a different runtime error
 
     # test_nn.py
     # TestNNDeviceType
-    'test_embedding_backward',  # uses sparse
+    'test_embedding_backward',  # sparse
     'test_embedding_dense_grad',  # slow
-    'test_EmbeddingBag_per_sample_weights_and_new_offsets', # wrong result
-    'test_batchnorm_grad',
-    'test_gumbel_softmax_xla',
-    'test_rnn_retain_variables',
-    'test_pool_invalid_size',  # doesn't raise
-    'test_nonlinearity_propagate_nan',
+    'test_EmbeddingBag_per_sample_weights_and_new_offsets', # UndefinedTensorImpl::_singleton
+    'test_batchnorm_grad', # UndefinedTensorImpl::_singleton
+    'test_pool_invalid_size',  # expecting a different runtime error
+    'test_nonlinearity_propagate_nan', # relu6 with a nan tensor returns a tensor([0.]) instead of a nan tensor
     'test_InstanceNorm3d_general',  # precision (1e-2)
     'test_InstanceNorm2d_general',  # precision (1e-2)
     'test_InstanceNorm1d_general',  # precision (1e-2)
-    'test_EmbeddingBag_per_sample_weights_failures',  # runtime error
-    'test_variable_sequence',  # value error
-    'test_embedding_bag_device',  # runtime error
-    'test_embedding_bag_device',  # runtime error
-    'test_batchnorm_eval',  # runtime error
+    'test_EmbeddingBag_per_sample_weights_failures',  # expecting a different runtime error
+    'test_variable_sequence',  # PackedSequence batch_sizes.device.type should be CPU but is XLA
```

Comment:
This is related to an issue @mruberry used to look at, I think we'll likely need to workaround this instead of leaving it skipped.

Reply:
Pack should be CPU and unpack device, so if the test does otherwise, and we need to add code just to make the test happy, maybe we can live it disabled.